testcases/OpTestLPM.py

```diff
@@ -158,16 +159,51 @@ def vnic_options(self, remote=''):
 
         return " -i \"vnic_mappings=%s\" " % ",".join(cmd)
 
+    def is_RMC_active(self, mg_system):
```

Comment:
can we have function name like `is_RMCActive`

Reply:
Done